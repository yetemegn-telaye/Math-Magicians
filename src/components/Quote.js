import React from 'react';
import Header from './Header';
import '../styles/quote.css';

const Quote = () => (
  <>
    <Header />
    <div className="quote-body">
      <h4>
        Mathematics is not about numbers, equations, computations, or algorithms:
        it is about understanding. -William Paul Thurston
      </h4>
    </div>
  </>
);

export default Quote;
