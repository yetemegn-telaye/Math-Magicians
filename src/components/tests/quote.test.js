import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';
import Quote from '../Quote';

afterEach(() => {
  cleanup();
});

test('Quote component rendered', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
