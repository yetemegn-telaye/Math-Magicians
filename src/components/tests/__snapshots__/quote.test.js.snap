// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component rendered 1`] = `
<div
  className="quote-body"
>
  <h4>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston
  </h4>
</div>
`;
